packages/nvidia_nat_core/src/nat/tool/chat_completion.py

```
@@ -114,17 +118,25 @@ async def _chat_completion(chat_request_or_message: ChatRequestOrMessage) -> Cha
+            # other internal details in the user-facing response: the response
+            # is returned to the caller (potentially over the network) and may
+            # surface stack frames, DB schemas, API endpoints, or file paths
+            # that constitute an information-disclosure risk.
+            logger.exception("chat completion failed")
             last_content = ""
             try:
                 msg = GlobalTypeConverter.get().convert(chat_request_or_message, to_type=ChatRequest)
                 if msg.messages:
                     last = msg.messages[-1].content
                     last_content = last if isinstance(last, str) else str(last)
             except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)